obspy/core/tests/test_event.py

```diff
@@ -171,7 +171,7 @@ def test_farfield_2xn_input(self):
                         -1.13501984, 0.873480164], [0, 0, -0, 0, -0, 0],
                         [2.245, 0.655304008, 0.504304008, -2.245,
                          -0.655304008, -0.504304008]])
-        np.testing.assert_allclose(result, ref)
+        np.testing.assert_array_almost_equal(result, ref, decimal=5)
```

Comment:
Better use `assert_allclose()` with `atol` and `rtol` settings. The `assert_array_almost_equal` is not really consistent for floating point comparisions. `rtol=1E-5, atol=1E-8` should be pretty equivalent settings.